submissions/OlexiyDobroskok/ClassikFroggerGame/app.js

```diff
@@ -0,0 +1,169 @@
+const PLAYER_START_POSITION_X = 202;
+const PLAYER_START_POSITION_Y = 373;
+const PLAYER_STEP_X = 101;
+const PLAYER_STEP_Y = 83;
+const ENEMY_WIDTH = 171;
+const ENEMY_START_SPEED = 100;
+const ENEMIES_START_POSITION_Y = [43, 126, 209];
+const CANVAS_WIDTH = 505;
+const DELTA_WIDTH = 85;
+const DELTA_HEIGH = 3;
+
+const skins = [
+  {
+    id: "boy",
+    img: "images/char-boy.png",
+  },
+  {
+    id: "cat-girl",
+    img: "images/char-cat-girl.png",
+  },
+  {
+    id: "horn-girl",
+    img: "images/char-horn-girl.png",
+  },
+  {
+    id: "pink-girl",
+    img: "images/char-pink-girl.png",
+  },
+  {
+    id: "princess-girl",
+    img: "images/char-princess-girl.png",
+  },
+];
+
+const body = document.querySelector("body");
+const skinsList = document.createElement("ul");
+skinsList.classList.add("skins__list");
+body.prepend(skinsList);
+skinsList.prepend(...skinsListCreator());
+
+let score = 0;
+
+const result = document.createElement("p");
+result.classList.add("show__result");
+skinsList.after(result);
+result.innerHTML = `Score: ${score}`;
+
+function skinsListCreator() {
+  return skins.map((playerSkin) => {
+    const { id, img } = playerSkin;
+    const listItem = document.createElement("li");
+    listItem.classList.add("list__item");
+    listItem.innerHTML = id;
+    const skinImg = document.createElement("img");
+    skinImg.classList.add("list__image");
+    skinImg.src = img;
+    skinImg.id = id;
+    listItem.prepend(skinImg);
+    return listItem;
+  });
+}
+
+let playerSprite = "images/char-cat-girl.png";
+
+skinsList.addEventListener("click", function ({ target }) {
+  skins.forEach((skin) => {
```

Comment:
This is better done with `Array.find()`.

Reply:
Good advice,Thank you!)